pyrasterframes/src/main/python/pyrasterframes/__init__.py

```diff
@@ -167,10 +173,32 @@ def temp_name():
         .load(path, **options)
 
 
-def _aliased_writer(df_writer, format_key, path, **options):
-    """ Saves the dataframe to a file of the given type at the given path."""
-    return df_writer.format(format_key).save(path=path, options=options)
+def _geotiff_writer(
+    df_writer,
+    path=None,
+    crs=None,
+    raster_dimensions=None,
+    **options):
+
+    if raster_dimensions is not None:
+        if isinstance(raster_dimensions, tuple):
+            options.update({
+                "imageWidth": raster_dimensions[0],
+                "imageHeight": raster_dimensions[1]
+            })
+        elif isinstance(raster_dimensions, str):
+            parts = raster_dimensions.split(',')
+            options.update({
+                "imageWidth": parts[0],
```

Comment:
again maybe makes sense to check `len(parts)==2` and `all([isinstance(p, int) for p in parts])`